_source/handbook/03_page_refreshes.md

```diff
@@ -52,8 +52,7 @@ Sometimes, you want to ignore certain elements while morphing. For example, you
 You can use [turbo frames](/handbook/frames.html) to define regions in your screen that will get reloaded using morphing when a page refresh happens. To do so, you must flag those frames with `refresh="morph"`.
 
 ```html
-<turbo-frame id="my-frame" refresh="morph">
-  ...
+<turbo-frame id="my-frame" refresh="morph" src="/my_frame">
 </turbo-frame>
```

Comment:
`refresh="morph"` has no effect when there is no src on the turbo-frame. Updating this example so it's consistent with the statements below:

"...you can load additional content that didn't arrive in the initial page load ..." and "When a page refresh happens, Turbo won't remove the frame contents; instead, it will reload the turbo frame ..."